openedx/core/lib/tests/test_xblock_utils.py

```diff
@@ -0,0 +1,195 @@
+"""
+Tests for xblock_utils.py
+"""
+import ddt
+import uuid
+from unittest import TestCase
+from xblock.fragment import Fragment
+from django.test.client import RequestFactory
+from xblock.core import XBlock
+from xmodule.x_module import XModule, XModuleDescriptor
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory, LibraryFactory
+from lms.djangoapps.lms_xblock.runtime import quote_slashes
+from opaque_keys.edx.keys import CourseKey
+from courseware.models import StudentModule
+from student.tests.factories import UserFactory
+
+from xmodule.modulestore.django import modulestore
+from xmodule.contentstore.django import contentstore
+from xmodule.modulestore import ModuleStoreEnum, ModuleStoreRead
+from xmodule.modulestore.xml_importer import import_course_from_xml
+from django.conf import settings
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+
+from openedx.core.lib.xblock_utils import (
+    wrap_fragment,
+    request_token,
+    wrap_xblock,
+    replace_jump_to_id_urls,
+    replace_course_urls,
+    replace_static_urls,
+    grade_histogram,
+    add_staff_markup
+)
+
+TEST_DATA_DIR = settings.COMMON_TEST_DATA_ROOT
+
+
+class TestXblockUtils(ModuleStoreTestCase):
```

Comment:
It would be great to test this against both modulestores. We have a pattern to do this using `ddt`. See, for example:

https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/management/commands/tests/test_create_course.py#L47-L70